src/SearchBar.ios.js

```diff
@@ -178,7 +178,7 @@ export default class SearchBar extends React.PureComponent {
     if (this.props.onCancelPress) {
       this.props.onCancelPress(this.props.navigation.goBack);
     } else {
-      this.props.navigation.goBack();
```

Comment:
i think we should probably just let the user override it with `onCancelPress` rather than default to `goBack(null)` here

Reply:
reverted goBack(null) to goBack().